packages/shared/src/components/post/PostSourceInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React, { ReactElement } from 'react';
+import { Source } from '../../graphql/sources';
+import { Separator } from '../cards/common';
+import SourceButton from '../cards/SourceButton';
+import { ProfileImageSize } from '../ProfilePicture';
+
+interface SourceInfoProps {
+  source: Source;
+  date?: string;
+  size?: ProfileImageSize;
+  typo?: string;
+}
+
```

Comment:
Clicking the link arrow on the shared post modal header makes it “download” 
![7C6D5C53-2DDE-4488-A524-662E7C66F520](https://user-images.githubusercontent.com/1488164/215164128-d9f78863-e55b-4d66-8ccb-ef6c95dd9517.jpeg)
![BF9268C4-AEC1-4117-A1A4-A5F1073F7FF4](https://user-images.githubusercontent.com/1488164/215164144-933e324e-adb2-4e20-84a4-92b20e147219.png)